reference/configuration/security.rst

```diff
@@ -899,6 +899,54 @@ multiple firewalls, the "context" could actually be shared:
     ignored and you won't be able to authenticate on multiple firewalls at the
     same time.
 
+stateless
+~~~~~~~~~
+
+Firewalls can configure a ``stateless`` boolean option in order to declare that the session mustn't be used when authenticating user:
```

Comment:
it may be because i am not native speaker
But i do prefer « must not » instead of « mustn’t »